test/acceptance/README.md

```diff
@@ -1,204 +1,63 @@
-- `I expect that a (alertbox|confirmbox|prompt) is( not)* opened` <br>Check if a modal is opened
-- `I expect that a (alertbox|confirmbox|prompt)( not)* contains the text "$text"` <br>Check the text of a modal
-
-## When steps
+## Running tests in parallel
+Tests can be run in parallel at feature file level by increasing the number of instances available to WebdriverIO in `wdio.conf.js', e.g.:
+```js
+maxInstances: 3,
+```
 
-- `I (click|doubleclick) on the (link|button|element) "([^"]*)?"` <br>(Double)click a link, button or element
-- `I (add|set) "([^"]*)?" to the inputfield "([^"]*)?"` <br>Add or set the content of an input field
-- `I clear the inputfield "([^"]*)?"` <br>Clear an input field
-- `I drag element "([^"]*)?" to element "([^"]*)?"` <br>Drag an element to another element
-- `I submit the form "([^"]*)?"` <br>Submit a form
-- `I pause for (\d+)ms` <br>Pause for a certain number of milliseconds
-- `I set a cookie "([^"]*)?" with the content "([^"]*)?"` <br>Set the content of a cookie with the given name to  the given string
-- `I delete the cookie "([^"]*)?"` <br>Delete the cookie with the given name
-- `I press "([^"]*)?"` <br>Press a given key. You’ll find all supported characters [here](https://w3c.github.io/webdriver/webdriver-spec.html#keyboard-actions). To do that, the value has to correspond to a key from the table.
-- `I (accept|dismiss) the (alertbox|confirmbox|prompt)` <br>Accept or dismiss a modal window
-- `I enter "([^"]*)?" into the prompt` <br>Enter a given text into a modal prompt
-- `I scroll to element "([^"]*)?"` <br>Scroll to a given element
-- `I close the last opened (window|tab)` <br>Close the last opened browser window/tab
-- `I focus the last opened (window|tab)` <br>Focus the last opened browser window/tab
-- `I log in to site with username "([^"]*)?" and password "([^"]*)?"` <br>Login to a site with the given username and password
-- `I select the (\d+)(st|nd|rd|th) option for element "([^"]*)?"` <br>Select an option based on it's index
-- `I select the option with the (name|value|text) "([^"]*)?" for element "([^"]*)?"` <br>Select an option based on its name, value or visible text
-- `I move to element "([^"]*)?"( with an offset of (\d+),(\d+))` <br>Move the mouse by an (optional) offset of the specified element
+## Running tests against multiple browsers
+Tests can be run against multiple browsers, currently only when running outside a container, by specifying additional capabilities together with more instances in `wdio.conf.js`:
+```js
+maxInstances: 10,
+capabilities: [
+    { acceptInsecureCerts: true, browserName: 'chrome' },
+    { acceptInsecureCerts: true, browserName: 'firefox' },
+    { acceptInsecureCerts: true, browserName: 'edge' }
+],
+```
```

Comment:
Would it be worth adding a note for arm architectures re selenium image so that tests can run containerised for arm too? The change was from:
```
image: selenium/standalone-chrome
```
to:
```
image: seleniarm/standalone-chromium
```

Reply:
Good idea, will add to README and also as commented-out line in docker-compose.yaml